src/components/Payment/Payment.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from "react";
+import { GoogleLogin } from "react-google-login";
+import { Button } from "react-bootstrap";
+import { websocketHelper } from "../../api/WebsocketHelper";
+import { RequestType } from "../../api/ApiTypes.d";
+import { loadStripe } from "@stripe/stripe-js";
+import api from "../../api/ApiHelper";
+
+// After google sign in, ui does not refresh
+
+function Payment() {
+  let [premiumExpiration, setPremiumExpiration] = useState(new Date());
+
+  let [googleId, setGoogleId] = useState(localStorage.getItem('googleId'));
+
+  useEffect(() => {
```

Comment:
use useEffect(() => {}, [])
So it should only execute the first time (check with debugger or logs)